osu.Game.Rulesets.Catch/Skinning/Default/PulpFormation.cs

```diff
@@ -34,11 +33,23 @@ protected PulpFormation()
             distance * MathF.Sin(angle * MathF.PI / 180),
             distance * MathF.Cos(angle * MathF.PI / 180));
 
-        [BackgroundDependencyLoader]
-        private void load(DrawableHitObject drawableObject)
+        protected void Clear()
+        {
+            for (; numPulps > 0; numPulps--)
+                InternalChildren[numPulps - 1].Alpha = 0;
+        }
+
+        protected void Add(Vector2 position, Vector2 size)
         {
-            DrawableCatchHitObject drawableCatchObject = (DrawableCatchHitObject)drawableObject;
-            AccentColour.BindTo(drawableCatchObject.AccentColour);
+            if (numPulps == InternalChildren.Count)
```

Comment:
Not sure if it's just me but I got thoroughly confused by this check and how it can ever be false. I *think* the intent would be conveyed better if `numPulps` was renamed to `pulpsInUse`.